crates/p2p/src/tests/mod.rs

```diff
@@ -3,5 +3,7 @@ pub(crate) mod gossipsub;
 pub(crate) mod is_connected;
 pub(crate) mod new_op;
 
+#[cfg(feature = "quic")]
+pub(crate) mod quic;
 #[cfg(feature = "request-response")]
 pub(crate) mod request_response;
```

Comment:
great!

If you go with my `Cargo.toml` suggestion in the first comment of this review, can you also feature-gate `gossipsub`?